rotkehlchen/api/rest.py

```diff
@@ -3465,7 +3465,21 @@ def get_historical_assets_price(
             self,
             assets_timestamp: list[tuple[Asset, Timestamp]],
             target_asset: Asset,
+            only_cache_period: int | None = None,
     ) -> dict[str, Any]:
+        if only_cache_period is not None:
+            result: dict[str, Any] = {
+                'assets': defaultdict(lambda: defaultdict(lambda: ZERO_PRICE)),
+                'target_asset': target_asset.identifier,
+            }
+            for entry in GlobalDBHandler().get_historical_prices(
```

Comment:
it is a staticmethod

```suggestion
            for entry in GlobalDBHandler.get_historical_prices(
```